### DISCUSSION TITLE
Issue installing security onion. Anaconda network manager

### DISCUSSION THREAD

**Discussion by Ashc1121** [Security-Onion-Solutions/securityonion#10684] [General]
Looking for guidance on installing security onion. We are attempting to install the latest install 2.3.260. During the install we consistently run into an issue with Anaconda network manager (please see screenshot below). The localhost account that we created during the initial install does not take and we are unable to log in to finish the install.
 
Any possible guidance would be appreciated! 
 
 
![seconion](https://github.com/Security-Onion-Solutions/securityonion/assets/138228534/831b4044-c675-43f9-aa04-72a52235bc36)

**Comment by InfosecGoon** [comment]
Are you installing from an ISO? Did you check the hash on the downloaded ISO to make sure it wasn't corrupted in transit?

If you're installing by writing that ISO to a USB stick or a DVD, have you tried using different media?

**Reply by Ashc1121** [reply]
Hey, thank you for your response! Yes I checked the hash on the downloaded ISO. I have been trying to install it from the iDRAC remote console by attaching virtual media and then attaching the ISO. Do you think trying the install through a USB stick would be worthwhile? Thanks.

**Reply by InfosecGoon** [reply]
I would suggest trying it through the USB stick - when you attach the ISO via the DRAC there can be issues like this if the network connection between your workstation and the server experiences any hiccups.

**Reply by Ashc1121** [reply]
Thank you! Will certainly try that. Will let you know how it goes.
